.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# build project
+# e.g.
+# ./autogen.sh
+# ./configure
+# make -j$(nproc) all
+
+# build fuzzers
+# e.g.
+# $CXX $CXXFLAGS -std=c++11 -Iinclude \
+#     /path/to/name_of_fuzzer.cc -o $OUT/name_of_fuzzer \
+#     $LIB_FUZZING_ENGINE /path/to/library.a
+compile_go_fuzzer github.com/bomctl/bomctl/internal/pkg FuzzFetch Fuzz
```

Comment:
Does this need to be a script file? Seems like a one-liner in the Dockerfile should do the same thing. If so:

- Missing SPDX header
- Add standard shebang and `set` flags

```suggestion
#!/usr/bin/env bash

set -euo pipefail

compile_go_fuzzer github.com/bomctl/bomctl/internal/pkg FuzzFetch Fuzz
```